app/views/dump.py

```diff
@@ -0,0 +1,52 @@
+from flask_login import current_user
+from flask_login import login_required
+from flask import Blueprint, jsonify
+
+from app.globals import get_answer_store, get_metadata
+from app.submitter.converter import convert_answers
+from app.utilities.schema import get_schema
+from app.questionnaire.path_finder import PathFinder
+
+
+dump_blueprint = Blueprint('dump', __name__)
+
+
+@dump_blueprint.route('/dump/<store>', methods=['GET'])
+@login_required
+def dump_data(store):
+    response = {'status': 'Unknown', 'message': ''}
+    metadata = get_metadata(current_user)
+
+    if 'dumper' in metadata.get('roles', []):
+        answer_store = get_answer_store(current_user)
+
+        if answer_store.answers:
```

Comment:
Also we might want to submit answers for a survey which is completely optional (all questions optional)